bindings/nodejs/lib/types/client/network.ts

```diff
@@ -7,36 +7,47 @@ import { INodeInfoProtocol } from '../models/info';
  * Network types.
  */
 export enum Network {
+    /** The mainnet. */
     Mainnet,
+    /** The testnet */
     Testnet,
 }
 
 /**
  * Basic Auth or JWT.
  */
 export interface IAuth {
+    /** JWT authentication parameters. */
     jwt?: string;
+    /** Basic authentication parameters. */
     basicAuthNamePwd?: [string, string];
 }
 
 /**
  * Options for the MQTT broker.
  */
 export interface IMqttBrokerOptions {
+    /** Whether the MQTT broker should be automatically disconnected when all topics are unsubscribed or not. */
     automaticDisconnect?: boolean;
-    /** timeout in seconds */
+    /** Sets the timeout used for the MQTT operations. */
     timeout?: number;
+    /**Sets the use_ws used for the MQTT operations. */
     useWs?: boolean;
+    /**Sets the port used for the MQTT operations. */
```

Comment:
```suggestion
    /** Sets whether websockets should be used instead of regular TCP for the MQTT operations. */
    useWs?: boolean;
    /** Sets the port used for the MQTT operations. */
```